### DISCUSSION TITLE
No network access from WSL/Ubuntu

### DISCUSSION THREAD

**Discussion by sb5346** [microsoft/WSL#7131] [Q&A]
I have reinstalled WSL/Ubuntu 20.04 on my Windows10 computer and have been trying to install packages such as volta.sh, node etc.
However, I keep getting errors that suggest there is a problem with the network connection resolving.
I tried to ping 8.8.8.8 but get an error, network is unreachable.
I have checked the Windows features to make sure all necessary are switched on.
I have disabled my firewall to check whether that was the problem. Have gone through the checklist: https://docs.microsoft.com/en-us/windows/wsl/faq
But none seem to be resolving the problem.
Regards

**Comment by pieterhouwen** [comment]
Try the following:

```
echo nameserver 8.8.8.8 >/etc/resolv.conf
ping 8.8.8.8
```
If this works create a wsl.conf file in /etc:
```
[network]
generateresolvconf = false
```
